libs/core/src/schemas/events.schemas.ts

```diff
@@ -1,6 +1,13 @@
 import { z } from 'zod';
 
-export const ThreadCreated = z.object({ thread: z.string() });
+export const ThreadCreated = z.object({
+  threadId: z.string(),
+  creatorAddress: z.string(),
+});
+export const ThreadUpvoted = z.object({
```

Comment:
we can also reuse schemas... most thread related commands and events will share this (id, user address, ...)